usr/share/rear/backup/NETFS/default/400_create_include_exclude_files.sh

```diff
@@ -1,33 +1,86 @@
 
-# Backup all that is explicitly specified in BACKUP_PROG_INCLUDE:
-for backup_include_item in "${BACKUP_PROG_INCLUDE[@]}" ; do
-    test "$backup_include_item" && echo "$backup_include_item"
-done > $TMP_DIR/backup-include.txt
+# What to include in the backup:
+cat /dev/null > "$TMP_DIR/backup-includes.txt"
```

Comment:
I prefer explicit verbose code over compressed coding, cf.
"explain what they do, even if it makes them longer"
and "prefer readability over compression" in
https://github.com/rear/rear/wiki/Coding-Style#code-must-be-easy-to-read

For the (not so) fun of it:
More than 30 years ago when I programmed in COBOL
I learned that the hard way because in traditional COBOL
reserved words are written in all capitals so basically
much of the code shouts at one which totally contradicts
that almost invisible '.' terminator, cf.
"terminated with a period are a common source of bugs" in
https://en.wikipedia.org/wiki/COBOL#Scope_termination
Have (not so much) fun with finding right and wrong dots
in larger COBOL-IS-SHO[OU]TING-AT-YOU code :-(